onnx/__init__.py

```diff
@@ -186,6 +188,11 @@ def save_model(proto: Union[ModelProto, bytes], f: Union[IO[bytes], str], format
     '''
     if isinstance(proto, bytes):
         proto = _deserialize(proto, ModelProto())
+    if save_as_external_data is None:
+        # Save to external data automatically if the model is larger than
+        # ONNX_SAVING_AS_EXTERNAL_DATA_THRESHOLD (default 2GB)
+        threshold = int(os.getenv("ONNX_SAVING_AS_EXTERNAL_DATA_THRESHOLD", 2 ** 31))
```

Comment:
```suggestion
        threshold = int(os.getenv("ONNX_SAVING_AS_EXTERNAL_DATA_THRESHOLD", onnx.checker.MAXIMUM_PROTOBUF))
```
I made onnx.checker.MAXIMUM_PROTOBUF as 2000000000 for lower boundary to detect possible 2GB exceed error. It seems fine to use 2 ** 31 directly: https://github.com/protocolbuffers/protobuf/blob/450d24ca820750c5db5112a6f0b0c2efb9758021/python/google/protobuf/pyext/message.cc#L1693. Feel free to update the number for onnx.checker.MAXIMUM_PROTOBUF. Thanks!

Reply:
Thanks! I will update it soon.